htdocs/admin/propal.php

```diff
@@ -615,6 +615,19 @@
 print "</tr>";
 
 
+print '<tr class="oddeven">';
+print '<td>'.$langs->trans("ModifyMarginRates").'</td>';
+print '<td class="center">';
+if ($conf->use_javascript_ajax) {
+	print ajax_constantonoff('PROPALE_MODIFY_MARGIN_RATES');
```

Comment:
I don't think we need an option PROPALE_MODIFY_MARGIN_RATES for this as it is an "addition" feature (so low risk to break existing).
Can you remove this option and remove condition PROPALE_MODIFY_MARGIN_RATES (we imagine this condition is always true).

Reply:
As suggested, removed PROPALE_MODIFY_MARGIN_RATES as not needed.